Source/WebCore/dom/ElementData.cpp

```diff
@@ -154,8 +154,8 @@ UniqueElementData::UniqueElementData(const ShareableElementData& other)
 Ref<UniqueElementData> ElementData::makeUniqueCopy() const
 {
     if (isUnique())
```

Comment:
This would be better written (could be slightly more efficient) as:
```
if (auto* uniqueData = dynamicDowncast<const UniqueElementData>(*this))
    return return adoptRef(*new UniqueElementData(*uniqueData));
return adoptRef(*new UniqueElementData(downcast<const ShareableElementData>(*this)));
```

Otherwise, we're doing the `isUnique()` check twice.